config/initializers/curation_concerns.rb

```diff
@@ -9,3 +9,13 @@
   end
 end
 
+# We're actually overriding from the sufia-set Sufia::FileSetPresenter, to our
+# own. to_prepare seems necessary to make it stick in dev.
+Rails.application.config.to_prepare do
+  CurationConcerns::MemberPresenterFactory.file_presenter_class = CHF::FileSetPresenter
```

Comment:
Work for me! 

I don't really think of just setting a global variable as an "override" or "monkey patch".  it's more like a configuration setting. It does need the `to_prepare` because of Rails dev-mode reloading though.  But I can see it going both ways, works for me.